wandb/sdk/launch/_project_spec.py

```diff
@@ -254,7 +254,7 @@ def _fetch_project_local(self, internal_api: Api) -> None:
                     # need to rebuild image with new code
                     self.build_image = True
 
-            if entry_point.endswith("ipynb"):
+            if "_session_history.ipynb" in os.listdir(self.project_dir):
```

Comment:
technically with the changes in jupyter.py this is no longer needed, since colab nb entry points should end in .ipynb